.github/CODEOWNERS

```diff
@@ -353,7 +353,7 @@ packages/deeplinks/devtools @elastic/kibana-management
 packages/deeplinks/fleet @elastic/fleet
 packages/deeplinks/management @elastic/kibana-management
 packages/deeplinks/ml @elastic/ml-ui
-packages/deeplinks/observability @elastic/obs-ux-logs-team
+packages/deeplinks/observability @elastic/obs-ux-logs-team @elastic/obs-ux-management-team
```

Comment:
What is this package? I expected this change to be for the onboarding plugin in observability_solution...

Reply:
My bad @jasonrhodes , seems like I updated the wrong file by accident. Should be fixed now. Thanks for noticing!

Reply:
@jasonrhodes is this ok now?

Reply:
I'm meeting with @gbamparop and @flash1293 tomorrow to clarify things a bit and then I'll come back here and approve this.